fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -12,11 +12,34 @@ def self.log
 
     # Runs a given command using backticks (`)
     # and prints them out using the UI.command method
-    def self.backticks(command, print: true)
-      UI.command(command) if print
-      result = `#{command}`
-      UI.command_output(result) if print
-      return result
+    def self.backticks(command, print: true, retriable_options: {})
+      require 'retriable'
```

Comment:
Let's wait for @mfurtak's opinion first on if we want to add the dependency, before updating the PR, sorry for chiming it like this